docs/core/tools/dotnet-tool-search.md

```diff
@@ -0,0 +1,95 @@
+---
+title: dotnet tool search command
+description: The dotnet tool search command searches the .NET Core tools that are published to NuGet.org.
+ms.date: 11/11/2020
+---
+# dotnet tool search
+
+**This article applies to:** ✔️ .NET 5.0 SDK and later versions
+
+## Name
+
+`dotnet tool search` - searches all [.NET Core tools](global-tools.md) that are published to NuGet.
+
+## Synopsis
+
+```dotnetcli
+dotnet tool search --detail  --prerelease
+    --skip <NUMBER> --take <NUMBER> <SEARCH TERM>
+
+dotnet tool search -h|--help
+```
+
+## Description
+
+The `dotnet tool search` command provides a way for you to search NuGet for tools that can be used as .NET global, tool-path, or local tools. The command searches the tool names and metadata such as titles, descriptions, and tags.
+
+The command uses the [NuGet Search API](https://docs.microsoft.com/nuget/api/search-query-service-resource#search-for-packages). It filters on `packageType=dotnettool` to select only .NET tool packages.
+
+## Options
+
+- **`--detail`**
+
+  Shows detailed results from the query.
+
+- **`--prerelease`**
+
+  Includes pre-release packages.
+
+- **`--skip <NUMBER>`**
+
+  Specifies the number of query results to skip. Used for pagination.
+
+- **`--take <NUMBER>`**
+
+  Specifies the number of query results to show. Used for pagination.
+
+- **`-h|--help`**
+
+  Shows command-line help.
+
+## Examples
+
+- **`dotnet tool search format`**
+
+  Searches NuGet.org for .NET tools that have "format" in their package name or description.
+
+  The output looks like the following example:
+
+  ```ouput
+  Package ID                              Latest Version      Authors                                                                     Downloads      Verified
+  ---------------------------------------------------------------------------------------------------------------------------------------------------------------
+  dotnet-format                           4.1.131201          Microsoft                                                                   496746
+  bsoa.generator                          1.0.0               Microsoft                                                                   533
+  ```
```

Comment:
In other articles, we put the example statement at the beginning rather than the command. Like this 

-   Search NuGet.org for .NET tools that have "format" in their package name or description

   ```dotnetcli
   dotnet tool search format
   ```

Reply:
I started using this format in the dotnet tool docs as it better parallels the presentation of options and with the intention of gradually moving in that direction in the other docs. But now that you mention it, I'm not sure it's worth the effort of changing all the others and will go back to the descriptive-text-first format.